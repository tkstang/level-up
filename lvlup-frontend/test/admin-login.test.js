import AdminLogin from '../src/components/Admin/login/login';
import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import thunk from 'redux-thunk';
import configureMockStore from 'redux-mock-store';
import lvlupApp from '../src/reducers/index';

const middlewares = [thunk];
const mockStore = configureMockStore(middlewares);

test('AdminLogin should render', () => {
  const store = mockStore({ lvlupApp });
  const edit = shallow(
    <AdminLogin adminLoginInfo={{ status: false }} store={store} />,
  );
  expect(shallowToJson(edit)).toMatchSnapshot();
});
