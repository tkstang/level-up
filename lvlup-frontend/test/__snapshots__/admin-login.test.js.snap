exports[`test AdminLogin should render 1`] = `
<div>
  <div
    className="login">
    <Connect(ReduxForm) />
    <p
      id="need-account">
      Need an account?
      <Link
        replace={false}
        to="/signup-admin">
         Sign Up
      </Link>
      . Valid Galvanize email required. 
    </p>
  </div>
</div>
`;
